data/CT/incentives.json

```diff
@@ -153,7 +155,8 @@
       "homeowner"
     ],
     "short_description": {
-      "en": "Free energy assessment, with possible weatherization followups."
+      "en": "Free energy assessment, with possible weatherization followups",
```

Comment:
nit: consistently, these short descriptions have punctuation. Add period to end?

Reply:
Fixed in the spreadsheet and in the JSON.